db/migrations/000001_create_users_table.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE users;
```

Comment:
![image](https://github.com/user-attachments/assets/d4f7192b-227c-4415-bf1c-7b943fb7b63a)

Reply:
😆😆😆
"We regret to inform you that we have to shutdown this product..."